packages/maskbook/src/plugins/ITO/UI/CompositionDialog.tsx

```diff
@@ -32,6 +32,9 @@ export function CompositionDialog(props: CompositionDialogProps) {
     const account = useAccount()
     const chainId = useChainId()
 
+    //#region tabs
+    const state = useState(0)
```

Comment:
Maybe you should limit the value domain by `useState<0 | 1 | 2>(0)` etc. give them a meaningful name is better